.travis.yml

```diff
@@ -1,5 +1,5 @@
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
It can invoke docker but the builds themselves can no longer run inside a docker container.
This PR is a suggestion to solve the timeouts we were getting ... it would be a temporary fix if we merge it at all.

One question is how much slower if at all are the tests going to be if they run in Travis's normal systems as opposed inside their docker containers?